integrationtest/src/test/java/org/mapstruct/itest/testutil/runner/ProcessorSuiteRunner.java

```diff
@@ -43,11 +40,16 @@
 import org.junit.runner.notification.StoppedByUserException;
 import org.junit.runners.ParentRunner;
 import org.junit.runners.model.InitializationError;
+import org.mapstruct.itest.testutil.runner.ProcessorSuite.CommandLineEnhancer;
 import org.mapstruct.itest.testutil.runner.ProcessorSuite.ProcessorType;
 import org.mapstruct.itest.testutil.runner.ProcessorSuiteRunner.ProcessorTestCase;
 import org.mapstruct.itest.testutil.runner.xml.Toolchains;
 import org.mapstruct.itest.testutil.runner.xml.Toolchains.ProviderDescription;
 
+import static org.apache.maven.it.util.ResourceExtractor.extractResourceToDestination;
```

Comment:
Could we try and keep the static imports first? It's the default in Eclipse and I'd prefer to keep it that way to avoid unneccessary line changes.